.github/workflows/deploy-demo-staging.yaml

```diff
@@ -0,0 +1,69 @@
+name: Deploy to staging demo
+on:
+  push:
+    tags:
+      - 3.3**
```

Comment:
Demo staging is on 3.4 right now. Please rebase if necessary

Reply:
What a mess